is-wds-admin.php

```diff
@@ -84,20 +87,40 @@ public static function get_instance() {
 	}
 
 	/**
-	 * Sets up our plugin
+	 * Sets up our plugin.
 	 *
 	 * @since  1.0.0
 	 */
 	protected function __construct() {
 		$this->basename        = plugin_basename( __FILE__ );
+		$this->privileged_user = 'wds_admin'; // This can be whatever user is the privileged user.
+		$this->version         = $this->get_version();
+
+		// Add capability if it does not exist for the given user.
+		add_action( 'init', array( $this, 'add_cap_if_not_exists' ) );
+	}
+
+	/**
+	 * Get the version of this plugin.
+	 *
+	 * @author Aubrey Portwood
+	 * @since 1.0.0
+	 *
+	 * @return string The Version value in the header.
+	 */
+	private function get_version() {
+		$headers = (object) get_file_data( __FILE__, array(
+			'version' => 'Version',
+		) );
 
-		// This can be whatever user is the privileged user.
-		$this->privileged_users = array( 'wds_admin' );
+		return $headers->version;
```

Comment:
I would say since the header value isn't controlled by anyone other than us (e.g. the developer), and that it doesn't get changed by anyone else, we should assume that we wouldn't put anything crazy there ourselves so it's safe. It's only shown on the plugins page, and that value is got via the same method we use here, so if they trust it, so should we?